app/src/main/assets/gpl_2.html

```diff
@@ -15,9 +15,9 @@
 </p>
 
 <pre>
-Copyright (C) 1989, 1991 Free Software Foundation, Inc.  
-51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
-
+Copyright (C) 1989, 1991 Free Software Foundation, Inc.<br/>
```

Comment:
Are those line breaks necessary? IMO there is no need to add them since the text is inside a `<pre>`.

Reply:
You are right, but I use `pre { white-space: pre-wrap; }` to keep the width of the body at 100% of the view. That's why I added the line breaks manually.